.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+name: CI Build
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - 'README.md'
+      - '.vscode/**'
+  pull_request:
```

Comment:
did not add `branches` here because it makes sense to run CI on any `pull_request` and not only on those that are opened against `master` branch